src/Parser/Html.php

```diff
@@ -66,57 +64,39 @@ public function __construct()
      *
      * @return array array of inserts, two indexes, insert and close
      */
-    public function splitInsertsOnNewLines($insert): array
+    public function splitInsertsOnNewLines($insert)
     {
         $inserts = [];
-
         if (preg_match("/[\n]{2,}/", $insert) !== 0) {
-            $splits = (preg_split("/[\n]{2,}/", $insert));
+            $splits = preg_split("/[\n]{2,}/", $insert);
             $i = 0;
-            foreach (preg_split("/[\n]{2,}/", $insert) as $match) {
 
+            foreach (preg_split("/[\n]{2,}/", $insert) as $match) {
                 $close = false;
-
                 $sub_inserts = $this->splitInsertsOnNewLine($match);
-
                 if (count($sub_inserts) > 0) {
                     if (count($sub_inserts) === 1) {
                         if ($i === 0 || $i !== count($splits) - 1) {
                             $close = true;
                         }
-
-                        $inserts[] = [
-                            'insert' => $sub_inserts[0]['insert'],
-                            'close' => $close,
-                            'new_line' => false,
-                            'pre_new_line' => false
-                        ];
+                        $inserts[] = ['insert' => $sub_inserts[0]['insert'], 'close' => $close, 'new_line' => false, 'pre_new_line' => false];
                     } else {
                         $count = count($sub_inserts);
                         $i = 0;
+
                         foreach ($sub_inserts as $sub_insert) {
-                            $inserts[] = [
-                                'insert' => $sub_insert['insert'],
-                                'close' => (($count - 1) === $i ? true : $sub_insert['close']),
-                                'new_line' => $sub_insert['new_line'],
-                                'pre_new_line' => $sub_insert['pre_new_line']
-                            ];
+                            $inserts[] = ['insert' => $sub_insert['insert'], 'close' => $count - 1 === $i ? true : $sub_insert['close'], 'new_line' => $sub_insert['new_line'], 'pre_new_line' => $sub_insert['pre_new_line']];
```

Comment:
Maybe nice to manually adjust this one back. I rather like the readability and assurance of operator priority. Idem a few lines before here.

Reply:
Although I agree that it would be nicer, it does very much feel like goldplating, especially because the semi short-term goal is to not require backporting to PHP 5.6 at all and just use the library directly. To expend effort in what is soon to be dead code seems like wasted time.

Reply:
True. I'm a bit afraid given that we only got started and will dive into it more and more, and there will be a few more back-ports given their development cycle.

But I agree with the pragmatic approach. Lets go and fix it once we bump into this.